src/app/GroupCreator.jsx

```diff
@@ -109,24 +119,17 @@ const styles = {
   },
   userListContainer: {
     marginTop: 5,
-    height: '100%',
-    minWidth: 400,
-    '@media (max-width: 1000px)': {
-      marginLeft: -20
-    },
-    display: 'flex',
-    width: '100%'
+    height: 900,
```

Comment:
there's a way to compute this at runtime. it's going to behave differently on different screen sizes, but it would be nice if the bottom of this container always stuck to the bottom of the window. I'd ask our resident Radium expert @colinmegill if he knows how to do this.